src/librustdoc/html/static/rustdoc.css

```
@@ -390,17 +390,13 @@ nav.sub {
 	cursor: pointer;
 }
 
+.docblock-short {
+	word-wrap: anywhere;
```

`word-wrap` is the non-standard name for that property. It works as an alias in most browsers for `overflow-wrap`.
```suggestion
	overflow-wrap: anywhere;
```